crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -119,9 +119,14 @@ CREATE INDEX idx_nullifiers_prefix ON nullifiers(nullifier_prefix);
 CREATE INDEX idx_nullifiers_block_num ON nullifiers(block_num);
 
 CREATE TABLE transactions (
-    transaction_id BLOB    NOT NULL,
-    account_id     BLOB    NOT NULL,
-    block_num      INTEGER NOT NULL,
+    transaction_id               BLOB    NOT NULL,
+    account_id                   BLOB    NOT NULL,
+    block_num                    INTEGER NOT NULL, -- Block number in which the transaction was included.
+    initial_state_commitment     BLOB    NOT NULL, -- State of the account before applying the transaction.
+    final_state_commitment       BLOB    NOT NULL, -- State of the account after applying the transaction.
+    input_notes                  BLOB    NOT NULL, -- Serialized vector with the Nullifier of the input notes.
+    output_notes                 BLOB    NOT NULL, -- Serialized vector with the NoteId of the output notes.
+    size_in_bytes                INTEGER NOT NULL, -- Size of the row in bytes.
```

Comment:
I would explain that this is an estimated size taking into account size of output note records.